application/modules/core/helpers/ContentBlockHelper.php

```diff
@@ -0,0 +1,197 @@
+<?php
+namespace app\modules\core\helpers;
+
+use app\modules\core\models\ContentBlock;
+use devgroup\TagDependencyHelper\ActiveRecordHelper;
+use yii\caching\TagDependency;
+use yii\helpers\ArrayHelper;
+use yii;
+use app;
+
+class ContentBlockHelper
+{
+    public static $prefix = '[';
+    public static $suffix = ']';
+    private static $chunksByKey = [];
+    /**
+     * Compiles content string by injecting chunks into content
+     * @param  {string} $content     Original content with chunk calls
+     * @param  {string} $content_key Key for caching compiled content version
+     * @param  {yii\caching\Dependency} $dependency  Cache dependency
+     * @return {string}              Compiled content with injected chunks
+     */
+    public static function compileContentString($content, $content_key, $dependency) {
+        self::preloadChunks();
+        $matches = [];
+        preg_match_all('%['.static::$prefix.']{2}([^\]\[]+)['.static::$suffix.']{2}%ui', $content, $matches);
```

Comment:
in this case it was beginning of symbol group, but it doesn't matter, i removed it